configs/testdata/valkyrie_config.test.yml

```diff
@@ -25,17 +25,20 @@ pam: # player account management
 providers:
   - name: Evolution # Name of provider
     url: "https://evo-url" # url used for gameLaunch
+    base_path: "/evolution"
     auth: # auth is specific to each provider check /provider/{providerName}/config.go
       casino_key: evo-casino-key # Some providers require a casino identifier if for example you have multiple casinos with the same provider
       api_key: evo-api-key
       casino_token: evo-casino-token
   - name: Red Tiger
     url: "https://rt-url"
+    base_path: "/redtiger"
     auth: # auth is specific to each provider
       api_key: rt-api-key
       recon_token: rt-recon-token
   - name: Caleta
     url: "https://caleta-url"
+    base_path: "/caleta"
```

Comment:
Should provider_base_path & operator_base_path be configured in this sample as well? Because they are not optional right?

Reply:
technically, they are optional. But yes, they can be added there for completion!

Reply:
Okay, perhaps they should tagged with omitempty in the struct then just to make it clear